scripts/constants.sh

```diff
@@ -34,17 +34,40 @@ else
     SUBNET_EVM_COMMIT="$(git --git-dir="$SUBNET_EVM_PATH/.git" rev-parse HEAD || :)"
 fi
 
+# Don't export them as they're used in the context of other calls
```

Comment:
I'm not too sure what this comment refers to, maybe mention what variables should not be exported? i.e. `AVALANCHE_VERSION`?